{
  "description": "\n Exceptions for generated client libraries.", 
  "examples": [], 
  "id": "gcloud.streaming.exceptions.httperror", 
  "methods": [
    {
      "examples": [], 
      "exceptions": [], 
      "id": "gcloud.streaming.exceptions.HttpError.__init__", 
      "name": "__init__", 
      "params": [], 
      "returns": [], 
      "source": "/home/travis/build/GoogleCloudPlatform/gcloud-python/.tox/json-docs/lib/python2.7/site-packages/gcloud/streaming/exceptions.py/gcloud/streaming/exceptions.py#L38", 
      "type": "instance"
    }, 
    {
      "examples": [], 
      "exceptions": [], 
      "id": "gcloud.streaming.exceptions.HttpError.from_response", 
      "name": "from_response", 
      "params": [
        {
          "description": " the response which returned the error", 
          "name": "http_response", 
          "nullable": null, 
          "optional": null, 
          "types": [
            "<a data-custom-type=\"gcloud/streaming/http_wrapper/response\">gcloud.streaming.http_wrapper.Response</a>"
          ]
        }
      ], 
      "returns": [
        {
          "types": [
            "HttpError"
          ]
        }
      ], 
      "source": "/home/travis/build/GoogleCloudPlatform/gcloud-python/.tox/json-docs/lib/python2.7/site-packages/gcloud/streaming/exceptions.py/gcloud/streaming/exceptions.py#L58", 
      "type": "instance"
    }
  ], 
  "name": "Httperror", 
  "source": "/home/travis/build/GoogleCloudPlatform/gcloud-python/.tox/json-docs/lib/python2.7/site-packages/gcloud/__init__.py/gcloud/__init__.py"
}